pages/focus-flow-main.html

```diff
@@ -63,6 +63,13 @@ <h1 class="logo"><span class="focus">focus</span><span class="flow">flow</span><
         </div>
       </div>
       <p class="begin-text">Let’s begin!</p>
+      <p
+        id="subject-selection-error"
+        class="error-message"
+        style="display: none; color: red; text-align: center"
+      >
+        Please select at least 2 subjects before proceeding.
```

Comment:
Put the CSS part in https://github.com/SagarGi/focusflow/blob/main/css/focus-flow-main.css. would be easy to find since it is for the main page.